### PR TITLE
doc: rewrite /docs/tools/util-linux.md

### PR DESCRIPTION
**Pull request by zdover23**
This PR refines the English language in
tools/util-linux.md. It also updates a link
to the GitHub repository that hosts the
util-linux project.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/tools/util-linux.md [hunk 1]
```diff
@@ -6,24 +6,24 @@ sidebar_label: Linux System Utilities
...
 
-As defined by the project itself, *util-linux* is a random collection of
-Linux&reg; utilities. This project is hosted
-on <a href="https://github.com/karelzak/util-linux" target="_blank">GitHub</a>.
-This project generally packaged in most distributions under the name
-*util-linux* and installed by default.
+*util-linux* is, as the documentation of the *util-linux* project itself says, 
+a random collection of Linux&reg; utilities. The *util-linux* project is hosted
+<a href="https://github.com/karelzak/util-linux" target="_blank">here on GitHub</a> or <a href="https://github.com/util-linux/util-linux" target=_blank">here on Github</a>. This project is packaged in most distributions under the name
+*util-linux* and is installed by default.
```
Comment (damien-lemoal) [inline docs/tools/util-linux.md:12]: Why the line break so early ?

File: docs/tools/util-linux.md [hunk 2]
```diff
@@ -52,8 +52,8 @@ sdc      host-managed
 sdd      host-managed
 ```
 
-The output of *lsblk* can also be formatted as needed using the `-o` option. For
-instance, the following command will display block device names, size and zone
+The output of *lsblk* can be formatted as by using the `-o` option. For
```
Comment (damien-lemoal) [inline docs/tools/util-linux.md:55]: s/as by/by

File: docs/tools/util-linux.md [hunk 3]
```diff
@@ -70,14 +70,14 @@ sdd       13.7T host-managed
...
-the *sg_rep_zone* and *sg_reset_wp* utilities of the
-[*sg3utils*](../tools/sg3utils.md) project, *blkzone* relies on the kernel
-provided ZBD `ioctl()` interface to perform zone report and zone reset
-operations. SCSI commands are not issued directly to the device by *blkzone*.
+The *blkzone* command line utility lists (reports) the zones of a zoned block
+device and resets the write pointer of sequential zones. Unlike the
+*sg_rep_zone* and *sg_reset_wp* utilities of the
+[*sg3utils*](../tools/sg3utils.md) project, *blkzone* relies on the kernel-
+provided ZBD `ioctl()` interface to perform "zone report" and "zone reset"
+operations. *blkzone* does not issue SCSI commands directly to the device.
```
Comment (damien-lemoal) [inline docs/tools/util-linux.md:78]: Can you change "SCSI" into "SCSI, ATA or NVMe" ? It was not there, but we should have it.

File: docs/tools/util-linux.md [hunk 4]
```diff
@@ -113,13 +113,14 @@ For more details see blkzone(8).
 ```
 
 :::note
-The open, close and finish commands of *blkzone* are available with *util-linux*
-version 2.36 onward. The capacity command is available on the master branch.
+The open, close and finish commands of *blkzone* are available only in
+*util-linux* version 2.36 and later. The `capacity` command is available on
+the master branch.
```
Comment (damien-lemoal) [inline docs/tools/util-linux.md:118]: The capacity is available since version 2.37.

File: docs/tools/util-linux.md (hunk 1 above)
Comment (zdover23) [reply docs/tools/util-linux.md:12]: In this case, I thought that the long paragraph was really three paragraphs:

(1) What util-linux is.
(2) What util-linux does.
(3) What util-linux is particularly good for.

If you think that this should be squashed into the same long paragraph, I'll defer to you. I think breaking up information into paragraphs like this makes it easier for people to digest the information in them.

Comment (damien-lemoal) [reply docs/tools/util-linux.md:12]: I think it is fine as is. I was merely remarking that the lines:

project is packaged in most distributions under the name
*util-linux* and is installed by default.

Could be better packed, filling up to 80 char limit:

project is packaged in most distributions under the name *util-linux* and is 
installed by default.

Not a big deal :)

**Comment by damien-lemoal** [comment]
Applied. Thanks !

**Pull request merged**